src/renderer/features/fellowship-tasks/components/tasks/PromotionRetentionVoting.tsx

```diff
@@ -84,6 +103,30 @@ export const PromotionRetentionVoting = ({ referendum, tags, transaction, onRefe
             <Markdown>{evidence?.summary ?? ''}</Markdown>
           </FootnoteText>
           {labelConfig ? <Label variant={labelConfig.color}>{t(labelConfig.text)}</Label> : null}
+          <div className="flex gap-16 text-left">
+            {evidence?.githubInfo?.pullRequests ? (
+              <div className="w-15">
+                <span className="text-footnote text-text-secondary">
+                  {t('fellowship.tasks.task.promotionVoting.pullRequests')}
+                </span>
+                &nbsp;
+                <span className="bold">{evidence?.githubInfo?.pullRequests}</span>
+              </div>
+            ) : (
+              <Skeleton height="20px" width="60px" />
+            )}
+            {evidence?.githubInfo?.mergedPullRequests ? (
+              <div className="w-15">
+                <span className="text-footnote text-text-secondary">
+                  {t('fellowship.tasks.task.promotionVoting.mergedPullRequests')}
+                </span>
+                &nbsp;
+                <span className="bold">{evidence?.githubInfo?.mergedPullRequests}</span>
```

Comment:
As I remember we've never used `bold` because boldness is wired into dedicated `text-__variant__`